src/Microsoft.ServiceFabric.Actors/Generator/ManifestGenerator.cs

```diff
@@ -1425,6 +1597,16 @@ private string GetApplicationManifestFilePath()
                 return string.Empty;
             }
 
+            private string GetStartupServicesFilePath()

```

Comment:
Minor: You could just replace separate method with this:

"return !string.IsNullOrEmpty(this.Arguments.StartupServicesFilePath) ? this.Arguments.StartupServicesFilePath : string.Empty;"